src/core/utils.py

```diff
@@ -1,8 +1,15 @@
 from typing import Any
+from datetime import date, datetime
 
 from sqlalchemy import Table, select
 from sqlalchemy.orm import Session
 
-
 def if_exists(model_class: Table, field: str, value: Any, session: Session) -> bool:
-    return session.scalar(select(model_class).filter(getattr(model_class, field) == value))
\ No newline at end of file
+    return session.scalar(select(model_class).filter(getattr(model_class, field) == value))
+
+def datetime_encoder(obj):
```

Comment:
Brak typingu  w sygnaturze funkcji, brak typu na `obj` moze nie jest tak zla, bo raczej bedzie Any, to return type juz powinien miec type hint